com.unity.netcode.gameobjects/Editor/Simulator/NetworkEventsView.cs

```diff
@@ -19,13 +19,20 @@ public NetworkEventsView(INetworkEventsApi networkEventsApi)
             m_NetworkEventsApi = networkEventsApi;
             AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(UXML).CloneTree(this);
 
+            RegisterCallback<DetachFromPanelEvent>(OnDetachedFromPanel);
```

Comment:
Resurfacing comment I gave during group coding, would be a good idea to use the `LifecycleEvents` pattern we use in virtual projects to do both registration and deregistration. Registering events from within the constructor opens the possibility of a leak, because constructed objects are not necessarily added to a panel and therefor not necessarily removed.